### ISSUE TITLE
Import error with python 3.8

### ISSUE THREAD

**Issue by infwinston** [google/generative-ai-python#13]
Hi,

I got the below error when importing this package with the latest version (`0.1.0rc2`). 
Did I miss anything?

```
>>> import google.generativeai
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/conda/envs/chatbot/lib/python3.8/site-packages/google/generativeai/__init__.py", line 59, in <module>
    from google.generativeai import types
  File "/opt/conda/envs/chatbot/lib/python3.8/site-packages/google/generativeai/types/__init__.py", line 19, in <module>
    from google.generativeai.types.text_types import *
  File "/opt/conda/envs/chatbot/lib/python3.8/site-packages/google/generativeai/types/text_types.py", line 34, in <module>
    class Completion(abc.ABC):
  File "/opt/conda/envs/chatbot/lib/python3.8/site-packages/google/generativeai/types/text_types.py", line 49, in Completion
    filters: Optional[list[safety_types.ContentFilterDict]]
TypeError: 'type' object is not subscriptable
```

**Comment by infwinston** [comment]
Upgrading to Python 3.9 resolves this.

**Comment by MarkDaoust** [comment]
Oh, it looks like our testing didn't cover python 3.8.


I'll remove that from the supported versions (or add testing).

b/282074714

**Comment by ianarawjo** [comment]
Honestly looking at the bug itself, the issue is fixed if you switch `list` for `List` from `typing`. I went into the code and did this (on Python 3.8.8), and it fixed the issue entirely:

<img width="330" alt="Screen Shot 2023-05-19 at 9 45 11 PM" src="https://github.com/google/generative-ai-python/assets/5251713/ca5b3f1f-8b8c-4732-a037-7ed7044a86ca">

**Comment by markmcd** [comment]
That's right - it was also fixed in #16, so you can check out that commit & pip install locally if you like. We have [very few](https://pypistats.org/packages/google-generativeai) users on 3.8, so it reduces our maintenance costs significantly to support fewer versions. Unless there's a compelling reason to support additional versions, we will likely follow Colab's default runtime version.

Although, @MarkDaoust - we currently have a package in PyPI that claims to supports 3.8, but is broken. WDYT about a "point" release so that we at least don't break 3.8 installs?

**Comment by MarkDaoust** [comment]
Yeah, we could have been more careful about that. Sorry. 

We've taken 3.8 out of the list of supported releases, and we've added testing for the Python versions we do support.
